### ISSUE TITLE
The AxisSwap feature has failed again in ppsspp 1.7.1

### ISSUE THREAD

**Issue by InvRetroBoy** [hrydgard/ppsspp#18945]
### Game or games this happens in

UCAS-40198-God of War Chains of Olympus

### What area of the game / PPSSPP

This feature has  failed again. The problem is more serious than last time. This time, when the AxisSwap hotkey was pressed, the game would hang immediately.

My operating system is Batocera v39, SDL version is v2.30.1, GPU is Amlogic G31 and  driver version is Mesa 24.0.3.

I change the SDL version to v2.80.5, but this bug still exists.

### What should happen

Pressing the AxisSwap hotkey in the game.

### Platform

Linux / BSD

### Mobile device model or graphics card (GPU)

Amlogic G31

### PPSSPP version affected

v1.7.1

### Last working version

v1.7.0

### Graphics backend (3D API)

OpenGL / GLES

### Checklist

- [X] Test in the [latest git build](https://buildbot.orphis.net/ppsspp/) in case it's already fixed.
- [X] [Search for other reports](https://github.com/hrydgard/ppsspp/search?q=my+issue&type=issues) of the same issue.
- [ ] Try resetting settings or older versions and include if the issue is related.
- [ ] Try without any cheats and without loading any save states.
- [ ] Include logs or screenshots of issue.

**Comment by rooster85** [comment]
Hello, not OP but just wanted to add that this also happens to me on my three devices I tested: 

Anbernic RG35XX plus device (Linux/Batocera V40, Mali-G31 MP2 GPU, opengl backend) 

android phone (Realme GT2 Pro, Android 13, Vulkan backend, snapdragon 8gen1, Adreno 730)

Odin pro handheld (Android 10, Vulkan backend, snapdragon 845, Adreno 630)

All three devices have PPSSPP 1.17.1 installed and the issue happens in multiple games/doesn't seem to be game-dependent. Tested in Final Fantasy reishiki and in Tekken dark resurrection on each device. 

Pressing mapped Axis swap button hangs PPSSPP each time. Tried mapping different buttons - L2, R2, L3, R3, always making sure the mapped button (not a combo) is not mapped to anything else. 
Thanks
